Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -121,25 +127,27 @@ ICompilationStatusNotificationPublisher notificationPublisher
       TextDocumentItem textDocument,
       DiagnosticErrorReporter errorReporter,
       Dafny.Program program,
-      bool loadCanceled
+      bool loadCanceled,
+      ILogger logger
     ) {
       return new DafnyDocument(
         textDocument,
         errorReporter,
         Array.Empty<Diagnostic>(),
         program,
-        CreateEmptySymbolTable(program),
+        CreateEmptySymbolTable(program, logger),
         loadCanceled
       );
     }
 
-    private static SymbolTable CreateEmptySymbolTable(Dafny.Program program) {
+    private static SymbolTable CreateEmptySymbolTable(Dafny.Program? program, ILogger logger) {
```

Comment:
It's not necessary to make `program` nullable here. This should resolve the nullability warnings.